html/cross-origin-embedder-policy/coep-frame-javascript.https.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<meta charset=utf-8>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/get-host-info.sub.js"></script>
+<script src="resources/script-factory.js"></script>
+<div id=log></div>
+<script>
+async_test(t => {
+  window.addEventListener("message", t.step_func_done(({ data }) => {
+    assert_equals(data.id, "");
+    assert_equals(data.origin, window.origin);
+    assert_true(data.sameOriginNoCORPSuccess);
+    assert_true(data.crossOriginNoCORPFailure, "Cross-origin without CORP did not fail");
+  }));
+  const frame = document.createElement("iframe");
+  t.add_cleanup(() => frame.remove());
+  frame.src = "resources/coep-frame.html";
+  frame.onload = t.step_func(() => {
+    frame.onload = null;
```

Comment:
Just out of curiosity:
I have seen tests that assume this automatically for popups. Do we need it explicitely here because it is an iframe or because it is a javascript url?

Reply:
(Talking about frame.onload = null)

Reply:
Some browsers dispatch the load event for javascript URLs (as is required) and others did not. So if you do not do this, you end up with a loop, at least in some browsers.